redaxo/src/core/lib/packages/null.php

```diff
@@ -181,7 +181,7 @@ public function getAuthor($default = null)
      */
     public function getVersion($format = null)
     {
-        return null;
+        return '';
```

Comment:
Die Version ist ein Pflichtfeld und wird bei Installation geprüft.
Deswegen mein Vorschlag, die Methode als nicht-nullable zu setzen, und hier im null-package halt einen leeren String zu liefern.
So richtig schön finde ich so leere Strings ja immer nicht, aber für mich überwiegen in diesem Fall die Vorteile.